pkg/cluster/resources.go

```diff
@@ -609,6 +610,44 @@ func (c *Cluster) createRoles() (err error) {
 	return c.syncRoles()
 }
 
+func (c *Cluster) createLogicalBackupJob() (err error) {
```

Comment:
I'm confused - why `createLogicalBackupJob` is defined in `cluster/resources`, `deleteLogicalBackupJob` in `cluster/cluster` and `syncLogicalBackupJob` in `cluster/sync`?

Reply:
this follows the implementation of  StatefulSets. I'll move `delete..` though as it does not seem to match